app/pages/Help.js

```diff
@@ -1,5 +1,6 @@
+import { th } from 'date-fns/locale';
```

Comment:
This is actually not used. Do you have some kind of auto import extension installed? I suggest to remove this line.

Reply:
done